### PR TITLE
Add 2 Syndicate Borg Shuttles

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

Adds two syndicate borg shuttles of similar design.
- One lone borg tasked with sabotaging/harassing the station. Being alone means they cannot heal themselves.
- Three borgs together tasked with sabotaging/harassing the station. On average you may expect one in three to be a Syndicate Saboteur borg which possesses a welder and may act as a "healer" for its team.

ID-Drone:
image

ID-Raider:
image

## Why / Balance

They are quite rare, but add some variety to shuttle events. I would describe them as somewhere between a dragon and a loneop in danger/effectiveness. They share the standard syndicate laws, and so may cause some emergent interactions with other round circumstances.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- add: 2 syndicate borg shuttles which may appear.

### CODE REVIEW

**Comment by lzk228** [comment]
your papers aren't locale supportable

**Comment by IProduceWidgets** [comment]
> your papers aren't locale supportable

I don't think we locale support mapped papers? Only prototypes like books etc. I mean I'm happy to do so, I just don't know if it makes sense to locale all these one-off things going forward.

**Comment by Morp9x** [comment]
The shuttle "Raider" looks disgusting, in my opinion. A huge big empty space. Maybe we should make the shuttle a little smaller?

**Comment by nymhole** [comment]
i played as a lone assault borg, i pretty much had to LOUDLY destroy every camera i saw, and kill anyone who saw me, since it went against the second directive. it was pretty fun and made for a cool mid round. there seems to be no use for the C4 stick though since the directives say to infiltrate undetected.

**Comment by ArtisticRoomba** [comment]
Definitely a really cool midround antag idea. When(TM) saboteur Borgs get fleshed out further this can lead to some exceptionally hilarious and chaotic gameplay (especially if RD just did nothing but crank out Borgs)

**Comment by TyAshley** [comment]
I am a big fan of this antagonist/event. I was able to play the Sabueteor borg (even without the Chameleon projector) on the Raider shuttle while it was still part of the main PR. The balancing of being stealthy while working toward your objectives with the two other borgs was very fun and also helped prevent general mayhem and murder. If too much murder/mayhem _is_ a concern, I would reword the exception handling to something like "Eliminate immediate threats as a last resort. Mission Integrity should not be compromised."

File: Resources/Maps/Shuttles/ShuttleEvent/drone.yml [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+        Directive 4: Disable all security systems.
+
+
+        Exception Handling:
+            Threat Detection: 
+                Engage only if mission integrity is compromised.
+            System Failure: 
+                Revert to secondary protocols.
+
+        01001001 00100000 01110100 01101000 01101001 01101110 01101011 00100000 01110100 01101000 01100101 01110010 01100101 01100110 01101111 01110010 00100000 01001001 00100000 01100001 01101101 [/bold][/color]
```
Comment (eoineoineoin) [inline Resources/Maps/Shuttles/ShuttleEvent/drone.yml:288]: > 01110100 01101000 01100101 01110010 01100101 01100110 01101111 01110010
Typo

**Comment by LankLTE** [comment]
We’ve opted to close this PR as we feel that while their objectives say to be stealthy there’s basically no mechanical prevention from them just going and attempting to kill the entire station, which the Borgs are very well equipped to do.
